data/hai/unfettered jobs.txt

```diff
@@ -349,8 +349,8 @@ mission "Unfettered Big Medicals Rush"
 		government "Hai"
 		attributes "spaceport"
 	to offer
-		has "Unfettered Jump Drive 1: offered"
-		random < 40
+		"reputation: Hai (Unfettered)" >= 0
+		random < 20 + "Unfettered Jump Drive 1: offered" * 20
```

Comment:
This is unrelated; why is this change being made?

Reply:
Because before this the only way to get positive relations was selling a JD

Reply:
OK